### PR TITLE
fix: user balance not updated after login #276

### LINKED ISSUE #276: bug: user balance not updated after login
the user balance in the rightbar next to the home page will not be updated after logging in. The balance in the personal information is updated normally.the balance is unequal.And Daily checkin bonus can be performed multiple times

### LINKED ISSUE #281: When logged in, changing language doesn't work
When logged in, after choosing English, it doesn't work

image

Still Chinese

image

### PR DESCRIPTION
**Pull request by oranges-eating**
Fix: https://github.com/casbin/casnode/issues/276

Fix: https://github.com/casbin/casnode/issues/281

demo:http://www.killerwing.xyz:7000
username:user_2
pw:123456
you can see that the score is updating at http://www.killerwing.xyz:7000/balance

### CODE REVIEW

**Comment by hsluoyz** [comment]
Don't call another API, get-account should work. @Kininaru what's your opinion?

File: web/src/main/BalanceBox.js [hunk 1]
```diff
@@ -366,7 +378,7 @@ class BalanceBox extends React.Component {
 
   render() {
     const { goldCount, silverCount, bronzeCount } = scoreConverter(
-      this.props.account?.score
+      this.state.member?.score
```
Comment (keainye) [inline web/src/main/BalanceBox.js]: `this.props.account` is same as `getMember -> this.state.member` here. So I think there is no need to `getMember` here.

**Comment by oranges-eating** [comment]
> 不要调用另一个 API，get-account 应该可以工作。@Kininaru你有什么看法？

getaccount is to obtain personal information directly from the session, which will not be updated

**Review by keainye — COMMENTED** [review]
I think this is not the problem of Casnode. 

https://github.com/casbin/casdoor/blob/master/object/user.go#L125
Here, Casdoor updated the information of the user except `Properties`. 
`balance` is a part of `Properties`, so it is not updated here. We should modify Casdoor to solve the problem instead of Casnode.

**Comment by keainye** [comment]
> getaccount is to obtain personal information directly from the session, which will not be updated

That's right. But to solve this problem, we'd better update session data after updating user.

**Comment by hsluoyz** [comment]
> > getaccount is to obtain personal information directly from the session, which will not be updated
> 
> That's right. But to solve this problem, we'd better update session data after updating user.

good point!

**Review by keainye — APPROVED** [review]

**Pull request merged**